container/tomcat-10.0/src/test/java/org/jboss/modcluster/container/tomcat/ContainerEventHandlerAdapterTestCase.java

```diff
@@ -45,16 +47,20 @@
 import org.mockito.Mockito;
 
 /**
+ * Test case for {@link TomcatEventHandlerAdapter}.
+ *
  * @author Paul Ferraro
+ * @author Radoslav Husar
  */
 public class ContainerEventHandlerAdapterTestCase {
+    protected final TomcatRegistry registry = mock(TomcatRegistry.class);
     protected final ContainerEventHandler eventHandler = mock(ContainerEventHandler.class);
     protected final LifecycleServer server = mock(LifecycleServer.class);
-    protected final TomcatFactory factory = mock(TomcatFactory.class);
+    protected final Object factory = mock(Object.class);
     protected final ServerProvider provider = mock(ServerProvider.class);
 
-    protected TomcatEventHandler createEventHandler(ContainerEventHandler eventHandler, ServerProvider provider, TomcatFactory factory) {
-        return new TomcatEventHandlerAdapter(eventHandler, provider, factory);
+    protected TomcatEventHandler createEventHandler(ContainerEventHandler eventHandler, ServerProvider provider, Object ignore) {
```

Comment:
Did you mean to leave this "ignore" parameter?

Reply:
Done.